scm/driver/harness/webhook.go

```diff
@@ -236,6 +257,34 @@ func convertPushHook(dst *pushHook) *scm.PushHook {
 	}
 }
 
+func convertPullRequestCommentHook(dst *pullRequestCommentHook) *scm.PullRequestCommentHook {
```

Comment:
**minor** maybe we shouldn't call the input `dst` (destination) when its actually the source of the convertion (we don't update it)